alert/alert.go

```diff
@@ -19,80 +20,77 @@ type Config struct {
 	RecipientEmails    []string
 	AWSAccessKeyID     string
 	AWSSecretAccessKey string
+
+	sesClient *ses.Client
 }
 
-func SendEmail(config Config, body string) {
-	if config.AWSAccessKeyID == "" || config.AWSSecretAccessKey == "" {
+func SendEmail(c Config, body string) {
+	if c.AWSAccessKeyID == "" || c.AWSSecretAccessKey == "" {
 		log.Printf("AWS credentials not provided for email alerts")
 		return
 	}
 
-	charSet := config.CharSet
-
-	subject := config.SubjectText
-	subjContent := ses.Content{
-		Charset: &charSet,
-		Data:    &subject,
-	}
-
-	msgContent := ses.Content{
-		Charset: &charSet,
-		Data:    &body,
+	if err := c.setSESClient(); err != nil {
```

Comment:
I moved this here since it seems like it doesn't need to constantly make a new client with every email, I could move it back to where it was.